app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -106,6 +107,18 @@ def check_enrollment(enrollment)
     status_check_attempted_at = Time.zone.now
     enrollment_outcomes[:enrollments_checked] += 1
 
+    if enrollment.profile_has_deactivation_reason?
+      log_enrollment_updated_analytics(
+        enrollment: enrollment,
+        enrollment_passed: false,
+        enrollment_completed: true,
+        response: nil,
+        reason: "Profile has a deactivation reason of #{enrollment.profile_deactivation_reason}",
```

Comment:
I think I am going to take a hybrid approach to this comment. I think it would be nice to keep the profile_deactivation_reason method in the enrollment model so that we can limit the amount of non-enrollment knowledge in the GetUspsProofingResultsJob.

so instead of this

```ruby
profile_deactivation_reason = enrollment.profile&.deactivation_reason
```

I will do this

```ruby
profile_deactivation_reason = enrollment.profile_deactivation_reason
```

Line 110 and 116 will be the same as you proposed

This also has the added benefit of not having to write tests for the enrollment not having a profile in the GetUspsProofingResultsJob spec.